### DISCUSSION TITLE
Any updates to yaml file or Annotations do not get reflected

### DISCUSSION THREAD

**Discussion by sroy19** [opensalt/opensalt#1334] [Q&A]
Hi everyone. 

I am new to Open salt and Symfony in general; although have used laravel exhaustively in the past.
I intended to put everything including /cfdoc behind firewall (login). Direct changes in method like indexAction where I check for log-in status and redirect works but it does not work via security.yaml configuration or by adding @Security annotation changes in the controller.

I made some small tweaks to the security.yaml file to see how the changes impact

For example:
core/config/packages/security.yaml
Replaced bottom:
  - { path: ^/, roles: IS_AUTHENTICATED_ANONYMOUSLY }
with
  - { path: ^/, roles: ROLE_SUPER_USER }

Or added Security annotation to 
core/src/Controller/Framework/LsDocController.php at the top.

I have no clue how to get it refreshed so as to reflect the security yaml or annotation changes. 
Even if I completely delete the security.yaml file or remove the annotations from existing Controllers, the changes have virtually no affect on pages and they load all just fine.

Ran cache clear - but no luck.
bin/console cache:clear
My .env file had APP_ENV set to prod and I changed it back to dev. Still no luck.

I am sure I am missing something pretty basic with annotation changes refresh/reload.
Your help is really appreciated!

**Comment by jmarks** [comment]
sroy19,

Can you tell us a bit about your use and if you plan to contribute code back? We are happy to provide support for contributors. If you are looking at a commercial use, PCG provides commercial support, maintenance and customization options..

**Reply by sroy19** [reply]
Absolutely happy to contribute back. Right now I am trying to learn CASE fundamentals, it's adaptation and usage. 
Seems interesting that you have a CASE testing app and wanted to try it out. 
The code am trying to adapt was to see if I could put all content behind firewall to begin with. This was so as to test out the security aspects of the tool.

Thanks in advance.

**Reply by jmarks** [reply]
Very good @sroy19 . Send an e-mail to jmarks@pcgus.com with some times this week and I will set up a quick intro with the lead developer and we can discuss. It might be faster then using issues. We will respond to your specific questions as we have time.

**Comment by roverwolf** [comment]
After changing the `security.yaml` file you will need to clear the cache to have it take effect.

If cache:clear is not working for some reason, you can try `rm -rf core/var/cache/{dev,prod}/*` (for the develop branch) to ensure the cache is completely empty.
